cmake/cmake_celix/BundlePackaging.cmake

```diff
@@ -189,10 +190,13 @@ function(add_celix_bundle)
     if (NOT DEFINED BUNDLE_SYMBOLIC_NAME)
         set(BUNDLE_SYMBOLIC_NAME ${BUNDLE_TARGET_NAME})
     endif ()
+
     if (NOT DEFINED BUNDLE_FILENAME)
-        set(BUNDLE_FILENAME ${BUNDLE_TARGET_NAME}.zip)
+        set(BUNDLE_FILENAME ${BUNDLE_TARGET_NAME})
     endif ()
 
+    set(BUNDLE_FILENAME ${BUNDLE_FILENAME}-${CMAKE_BUILD_TYPE}.zip)
```

Comment:
If would prefer that an postfix can be excluded for a configurable build type 

I took the liberty to create a diff for that:
```diff
diff --git a/cmake/cmake_celix/BundlePackaging.cmake b/cmake/cmake_celix/BundlePackaging.cmake
index b34fa138..3fb866c4 100644
--- a/cmake/cmake_celix/BundlePackaging.cmake
+++ b/cmake/cmake_celix/BundlePackaging.cmake
@@ -15,6 +15,9 @@
 # specific language governing permissions and limitations
 # under the License.
 
+
+set(CELIX_NO_POSTFIX_BUILD_TYPE "RelWithDebInfo" CACHE STRING "The build type used for creating bundle without a build type postfix.")
+
 find_program(JAR_COMMAND jar NO_CMAKE_FIND_ROOT_PATH)
 
 if(JAR_COMMAND AND NOT CELIX_USE_ZIP_INSTEAD_OF_JAR)
@@ -194,7 +197,11 @@ function(add_celix_bundle)
         set(BUNDLE_FILENAME ${BUNDLE_TARGET_NAME})
     endif ()
 
-    set(BUNDLE_FILENAME ${BUNDLE_FILENAME}-${CMAKE_BUILD_TYPE}.zip)
+    if ("${CMAKE_BUILD_TYPE}" STREQUAL "${CELIX_NO_POSTFIX_BUILD_TYPE}")
+        set(BUNDLE_FILENAME ${BUNDLE_FILENAME}.zip)
+    else ()
+        set(BUNDLE_FILENAME ${BUNDLE_FILENAME}-${CMAKE_BUILD_TYPE}.zip)
+    endif ()
 
     set(BUNDLE_FILE "${CMAKE_CURRENT_BINARY_DIR}/${BUNDLE_FILENAME}")
     #set(BUNDLE_CONTENT_DIR "${CMAKE_CURRENT_BINARY_DIR}/${BUNDLE_TARGET_NAME}_content")
diff --git a/libs/framework/gtest/CMakeLists.txt b/libs/framework/gtest/CMakeLists.txt
index ab94fff6..7777db49 100644
--- a/libs/framework/gtest/CMakeLists.txt
+++ b/libs/framework/gtest/CMakeLists.txt
@@ -42,6 +42,16 @@ add_executable(test_framework
 
 target_link_libraries(test_framework Celix::framework CURL::libcurl GTest::gtest)
 add_dependencies(test_framework simple_test_bundle1_bundle simple_test_bundle2_bundle simple_test_bundle3_bundle simple_test_bundle4_bundle simple_test_bundle5_bundle bundle_with_exception_bundle unresolveable_bundle_bundle)
+target_compile_definitions(test_framework PRIVATE
+        -DSIMPLE_TEST_BUNDLE1_LOCATION=\"$<TARGET_PROPERTY:simple_test_bundle1,BUNDLE_FILE>\"
+        -DSIMPLE_TEST_BUNDLE2_LOCATION=\"$<TARGET_PROPERTY:simple_test_bundle2,BUNDLE_FILE>\"
+        -DSIMPLE_TEST_BUNDLE3_LOCATION=\"$<TARGET_PROPERTY:simple_test_bundle3,BUNDLE_FILE>\"
+        -DSIMPLE_TEST_BUNDLE4_LOCATION=\"$<TARGET_PROPERTY:simple_test_bundle4,BUNDLE_FILENAME>\"
+        -DSIMPLE_TEST_BUNDLE5_LOCATION=\"$<TARGET_PROPERTY:simple_test_bundle5,BUNDLE_FILENAME>\"
+        -DTEST_BUNDLE_WITH_EXCEPTION_LOCATION=\"$<TARGET_PROPERTY:bundle_with_exception,BUNDLE_FILE>\"
+        -DTEST_BUNDLE_UNRESOLVEABLE_LOCATION=\"$<TARGET_PROPERTY:unresolveable_bundle,BUNDLE_FILE>\"
+)
+
 target_include_directories(test_framework PRIVATE ../src)
 
 configure_file(config.properties.in config.properties @ONLY)
diff --git a/libs/framework/gtest/src/bundle_context_bundles_tests.cpp b/libs/framework/gtest/src/bundle_context_bundles_tests.cpp
index d30d7eaa..dc16012c 100644
--- a/libs/framework/gtest/src/bundle_context_bundles_tests.cpp
+++ b/libs/framework/gtest/src/bundle_context_bundles_tests.cpp
@@ -35,13 +35,13 @@ public:
     celix_bundle_context_t *ctx = nullptr;
     celix_properties_t *properties = nullptr;
 
-    const char * const TEST_BND1_LOC = "simple_test_bundle1-" BUILD_TYPE ".zip";
-    const char * const TEST_BND2_LOC = "simple_test_bundle2-" BUILD_TYPE ".zip";
-    const char * const TEST_BND3_LOC = "simple_test_bundle3-" BUILD_TYPE ".zip";
-    const char * const TEST_BND4_LOC = "simple_test_bundle4-" BUILD_TYPE ".zip";
-    const char * const TEST_BND5_LOC = "simple_test_bundle5-" BUILD_TYPE ".zip";
-    const char * const TEST_BND_WITH_EXCEPTION_LOC = "bundle_with_exception-" BUILD_TYPE ".zip";
-    const char * const TEST_BND_UNRESOLVEABLE_LOC = "unresolveable_bundle-" BUILD_TYPE ".zip";
+    const char * const TEST_BND1_LOC = "" SIMPLE_TEST_BUNDLE1_LOCATION "";
+    const char * const TEST_BND2_LOC = "" SIMPLE_TEST_BUNDLE2_LOCATION "";
+    const char * const TEST_BND3_LOC = "" SIMPLE_TEST_BUNDLE3_LOCATION "";
+    const char * const TEST_BND4_LOC = "" SIMPLE_TEST_BUNDLE4_LOCATION "";
+    const char * const TEST_BND5_LOC = "" SIMPLE_TEST_BUNDLE5_LOCATION "";
+    const char * const TEST_BND_WITH_EXCEPTION_LOC = "" TEST_BUNDLE_WITH_EXCEPTION_LOCATION "";
+    const char * const TEST_BND_UNRESOLVEABLE_LOC = "" TEST_BUNDLE_UNRESOLVEABLE_LOCATION "";
 
     CelixBundleContextBundlesTests() {
         properties = properties_create();
```

Reply:
Add discussed, I pushed the changes for this.